sam/test-event.json

```diff
@@ -0,0 +1,42 @@
+{
+  "body": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFgR8x4W9X5Ix2tCx8YYB1MysVpSDj5kzAdsQJIC+iP0 paultag@nyx",
+  "path": "/hallow",
+  "httpMethod": "PUT",
+  "isBase64Encoded": false,
+  "headers": {
+    "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
+    "Accept-Encoding": "gzip, deflate, sdch",
+    "Accept-Language": "en-US,en;q=0.8",
+    "Cache-Control": "max-age=0",
+    "Host": "1234567890.execute-api.us-east-1.amazonaws.com",
+    "X-Forwarded-For": "127.0.0.1, 127.0.0.2",
+    "X-Forwarded-Port": "443",
+    "X-Forwarded-Proto": "https"
+  },
+  "requestContext": {
+    "accountId": "123456789012",
+    "resourceId": "123456",
+    "stage": "prod",
+    "requestId": "c6af9ac6-7b61-11e6-9a41-93e8deadbeef",
+    "requestTime": "09/Apr/2015:12:34:56 +0000",
```

Comment:
Oh my god, what is this format

Reply:
![](https://i.kym-cdn.com/photos/images/original/000/908/702/1ee.png)